Makefile.include

```diff
@@ -186,6 +186,7 @@ INCLUDES += -I$(RIOTBOARD)/$(BOARD)/include
 include $(RIOTBASE)/Makefile.pseudomodules
 include $(RIOTBASE)/Makefile.defaultmodules
 include $(RIOTBOARD)/$(BOARD)/Makefile.include
+-include $(RIOTBOARD)/$(BOARD)/Makefile.features
```

Comment:
no, the minus sign means to ignore the include (as opposed to abort with error) if the file is not found.